src/main/resources/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder/help-projectName.html

```diff
@@ -4,4 +4,5 @@
   demoproject <br />
   My FireOS Project <br />
   My Android Project <br />
+  In case you are not able to see the Projects refer to <a href = "https://github.com/awslabs/aws-device-farm-jenkins-plugin#project-section-not-being-populated-with-latest-data" target="_blank">troubleshooting guide</a>
```

Comment:
In case you are not able to see the Projects, refer to

Reply:
Changed.